Detectors/TRD/base/include/TRDBase/PadResponse.h

```diff
@@ -29,8 +29,8 @@ class PadResponse
  private:
   void samplePRF();                               // Initialized the PRF
   static constexpr int mPRFbin{500};              // Number of bins for the PRF
-  static constexpr float mPRFlo{-1.5};            // Lower boundary of the PRF
-  static constexpr float mPRFhi{1.5};             // Higher boundary of the PRF
+  const float mPRFlo{-1.5};                       // Lower boundary of the PRF
+  const float mPRFhi{1.5};                        // Higher boundary of the PRF
```

Comment:
out of curiosity, what's the reason for this change?

Reply:
I think there is no need for those variables to be static, so it is better to avoid it: https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=Global_Variables#Global_Variables

Reply:
Thanks for the comments.